Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,120 @@
+# Glossary
+
+## Ceph
+
+Ceph is a distributed network storage and file system with distributed metadata management and POSIX semantics. See also the [Ceph Glossary](https://docs.ceph.com/en/latest/glossary/) for more definitions.
+
+### MON
+
+Ceph Monitor is a daemon that maintains a map of the state of the cluster. This “cluster state” includes the monitor map, the manager map, the OSD map, and the CRUSH map. A Ceph cluster must contain a minimum of three running monitors in order to be both redundant and highly-available. Ceph monitors and the nodes on which they run are often referred to as “mon”s.
+
+### MGR
+
+The Ceph manager software, which collects all the state from the whole cluster in one place.
+
+### MDS
+
+The Ceph MetaData Server daemon. Also referred to as “ceph-mds”. The Ceph metadata server daemon must be running in any Ceph cluster that runs the CephFS file system. The MDS stores all filesystem metadata.
+
+### OSD
+
+Ceph Object Storage Daemon. The Ceph OSD software, which interacts with logical disks.
+
+### RBD
+
+The block storage component of Ceph. Also called "RADOS Block Device". It's a software instrument that orchestrates the storage of block-based data in Ceph.
+
+### RGW
+
+RADOS Gate Way. The component of Ceph that provides a gateway to both the Amazon S3 RESTful API and the OpenStack Swift API.
+
+## Kubernetes
+
+Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications. Here are some of the important terms from kubernetes perspective: 
```

Comment:
```suggestion
Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications. Here are some important terms related to kubernetes: 
```

Also, it could be helpul to add links to their definitions in the kubernetes glossary